contracts/tokenomics/generator/README.md

```diff
@@ -210,3 +211,44 @@ Returns orphan rewards amount.
   }
 }
 ```
+
+## Cw20HookMsg
+
+### `Deposit`
+
+Deposits given lp amount and allocates ASTRO.
+Must be sent from token contract.
```

Comment:
Documentation is not clear, please use following example, so users know how to use it:
https://docs.terraswap.io/docs/howto/freq_used_msg/#withdraw